.github/CONTRIBUTING.md

```diff
@@ -12,53 +11,44 @@ Support KB staff members review issues and pull requests on a regular basis. We
-1. Review the [Support KB guidelines below](#contribution-guidelines).
+* Make sure you have a [GitHub account](https://github.com/signup/free).
+* [Fork](https://help.github.com/articles/fork-a-repo/) the [Support KB repository](https://github.com/magento/knowledge-base). Remember to [sync your fork](https://help.github.com/articles/syncing-a-fork/) and update branches as needed.
+* Review the [Support KB guidelines below](#contribution-guidelines).
 
+>**Note for partners:** Add [2FA](https://devdocs.magento.com/contributor-guide/contributing.html#two-factor) protection when contributing to Magento repositories.
 
 ## Contribute
 
 The following diagram shows the contribution workflow:
 
 ![Contributing workflow](../docs/img/Contributor-PR.png)
 
-
 ### Create a branch
 
-1. Create a new branch from your fork using a name that best describes the work or references a GitHub issue number.
-1. Edit or create markdown (`.md`) files in your branch.
-1. Push your branch to your fork.
+* Create a new branch from your fork using a name that best describes the work or references a GitHub issue number.
+* Edit or create markdown (`.md`) files in your branch.
+* Push your branch to your fork.
 
 ### Create a pull request
 
-1. Create a pull request to the [magento/knowledge-base](https://github.com/magento/knowledge-base) repository. Use `main` as the base branch when creating a PR.
-
-1. Complete the pull request providing the information listed in the template.
+* Create a pull request to the [magento/knowledge-base](https://github.com/magento/knowledge-base) repository. Use `main` as the base branch when creating a PR.
+* Complete the pull request providing the information listed in the template.
+    * **We will close your pull request if you do not provide the information described in the template.**
+* After creating a pull request, a Support KB staff member will review it and may ask you to make revisions.
+    * **We will close your pull request if you do not respond to feedback in two weeks.**
 
-    **We will close your pull request if you do not provide the information described in the template.**
-
-1. After creating a pull request, a Support KB staff member will review it and may ask you to make revisions.
-
-    **We will close your pull request if you do not respond to feedback in two weeks.**
-
-**Note:** If you have not signed the [Adobe Contributor License Agreement](https://opensource.adobe.com/cla.html), the pull request provides a link. You must sign the CLA before we can accept your contribution.
+>**Note:** If you have not signed the [Adobe Contributor License Agreement](https://opensource.adobe.com/cla.html), the pull request provides a link. You must sign the CLA before we can accept your contribution.
 
```

Comment:
Same comment here, I think order is important here

Reply:
✔Applied suggested changes.
![bitmoji](https://sdk.bitmoji.com/render/panel/6e3eb687-79c8-443e-b053-fc574f6a5be9-d32433e1-8d42-44d9-9291-0f172baf2c26-v1.png?transparent=1&palette=1&width=246)